introduction.md

```diff
@@ -1,3 +1,6 @@
 Hi people!
 my favorite Code School path is JavaScript
 
+* Git
+* Ruby
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?